kubernetes/crd/dataset.json

```diff
@@ -0,0 +1,35 @@
+{
+	"apiVersion": "apiextensions.k8s.io/v1",
+	"kind": "CustomResourceDefinition",
+	"metadata": { "name": "datasets.new.artificialwisdom.cloud" },
+	"spec": {
+		"group": "new.artificialwisdom.cloud",
+		"versions": [{
+			"name": "v1",
+			"served": true,
+			"storage": true,
+			"schema": {
+				"openAPIV3Schema": {
+					"type": "object",
+					"properties": {
+						"spec": {
+							"type": "object",
+							"properties": {
+								"source": { "type": "string" },
+								"filename": { "type": "string" },
+								"bucket": { "type": "string" }
+							}
+						}
+					}
+				}
+			}
+		}],
+		"scope": "Namespaced",
+		"names": {
+			"plural": "datasets",
+			"singular": "dataset",
+			"kind": "DataSet",
```

Comment:
Dataset casing would be better.

Reply:
Yeah, it's a weird k8s convention that any hypenation point has to be capitalized like that. And I'm open to bikeshedding the name, too; in my notes, I called it `FeatherTable` because, well, it's a Feather table.